glean-core/rlb/tests/fog_init_fails.rs

```diff
@@ -0,0 +1,89 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! This test should be conceptually similar to how the RLB is used from e.g. FOG.
+//!
+//! We write a single test scenario per file to avoid any state keeping across runs
+//! (different files run as different processes).
+
+mod common;
+
+use std::{thread, time::Duration};
+
+use glean::Configuration;
+
+/// Some user metrics.
+mod metrics {
```

Comment:
The FOG* tests feel a bit out of place in here. Can we remove the references to FOG or move them in m-c?

Reply:
As above: I'd like to have these integration tests here. Much quicker to test _before_ we have to do the hard cycle of putting it into m-c, compiling for 20 minutes and then hope we trigger stuff there.

Reply:
Sure, but can we drop the `fog` word? It's confusing because there's nothing about FOG in this repo. These, to me, are simply integration tests for the RLB.